content/02.introduction.md

```diff
@@ -9,12 +9,20 @@ The low survival rates for some pediatric tumors are clearly multifactorial, exp
 Historically, some of the most fatal, inoperable brain tumors, such as diffuse midline gliomas, were not routinely biopsied due to perceived risks of biopsy and the paucity of therapeutic options that would require tissue. 
 Limited access to tissue to develop patient-derived cell lines and mouse models has been a barrier to research. 
 Furthermore, the incidence of any single brain tumor molecular subtype is relatively low due to the rarity of pediatric tumors in general. 
+
 To address these long-standing barriers, multiple national and international consortia have come together to collaboratively share specimens and data to accelerate breakthroughs and clinical translation.
+The creation of the Pediatric Brain Tumor Atlas (PBTA) in 2018, led by the Children's Brain Tumor Network (CBTN) and the Pacific Pediatric Neuro-Oncology Consortium (PNOC), was one such effort.
+Just as cooperation is required to share specimens and data, rigorous cancer genomic analysis requires collaboration among researchers with distinct expertise, such as computational scientists, bench scientists, clinicians, and pathologists.
+
+Although there has been significant progress in recent years to elucidate the landscape of somatic variation responsible for pediatric brain tumor formation and progression, translation of therapeutic agents to phase II or III clinical trials and subsequent FDA approvals have not kept pace.
+Within the last 20 years, the FDA has approved only five drugs for the treatment of pediatric brain tumors: mTOR inhibitor, everolimus, for subependymal giant cell astrocytoma [@doi:10.1097/01.COT.0000421359.88140.a8; @url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-everolimus-tuberous-sclerosis-complex-associated-partial-onset-seizures]; anti-PD-1 immunotherapy, pembrolizumab, for microsatellite instability–high or mismatch repair–deficient tumors [@url:https://www.fda.gov/drugs/drug-approvals-and-databases/fda-approves-pembrolizumab-adults-and-children-tmb-h-solid-tumors]; NTRK inhibitors larotrectinib [@url:https://www.fda.gov/drugs/fda-approves-larotrectinib-solid-tumors-ntrk-gene-fusions] and entrectinib [@url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-entrectinib-ntrk-solid-tumors-and-ros-1-nsclc] for tumors with an NTRK 1/2/3 gene fusions; MEK1/2 inhibitor, selumetinib, for neurofibromatosis type 1 (NF1) and symptomatic, inoperable plexiform neurofibromas [@url:https://www.fda.gov/drugs/resources-information-approved-drugs/fda-approves-selumetinib-neurofibromatosis-type-1-symptomatic-inoperable-plexiform-neurofibromas].
+<!--TODO: Now that this introduction is longer, would we consider using the same citations to refer to the five FDA-approved drugs without listing them all out?-->
```

Comment:
We've added to the intro without subtracting. Is this a place where we can subtract?

Reply:
I just added this TODO. My point is _not_ to cut "Within the last 20 years, the FDA has approved only five drugs for the treatment of pediatric brain tumors" which addresses your point in my opinion:

> I do _not_ think we should cut this section, as it is the major reason for requiring continued research on pediatric brain tumors (pediatric cancer in general) - we need more approved and effective therapies for these kids.

Is there anything in the rest of the paper that benefits from having the specific drugs and the labeled indications?

Reply:
Oh! We don't refer back to these, but we can in the discussion, since we bring up PD-1 and talk about MMR-deficient tumors, but generally if I was a reviewer, I would like to know if so few drugs were approved, what were they and it is good to not have to make the reader look up the many links to try to figure out 1) which cancers have had new therapies approved (notice no HGG/ATRT, etc - with the worst prognoses) and 2) what types of drugs have shown some efficacy, and can this work either provide more types of tumors which can be susceptible to those drugs or can we build upon this (eg if immunotherapy worked in X, maybe we can find another type of immunotherapy to work for another subtype or a related but not the same cancer type). I think I would feel differently if we had _many_ drugs approved and can link to some comprehensive website.

Reply:
To reiterate what the TODO says: You'd leave in the citations in that scenario, so someone who would want to examine the links could do that relatively easily.

I think part of my argument is that the reader (or reviewer) that would look this information up might already know some or most of that information because of their own domain expertise. 

If you don't know this information because of your own domain expertise, the list might not be enough to connect the dots re: "notice no HGG/ATRT" and your points about immunotherapy.

It could be at once too much and too little information depending on your background.

Reply:
>It could be at once too much and too little information depending on your background. 

:) I think you can say this about any introduction written in cancer? 

>If you don't know this information because of your own domain expertise, the list might not be enough to connect the dots re: "notice no HGG/ATRT" and your points about immunotherapy.

This is why I suggested we can bring this back to the discussion and connect more dots. 

This part was already reviewed by two neuro-onc clinicians (Peter Madsen on github and Cassie Kline via slack convo), so my suggestion would be that I can add more info to the discussion and we can see what the domain experts think. It is possible they may want even more biology or more dramatic calls to action - I tried to keep it concise while still being firm as I would in a grant, that this is an urgent research/clinical problem and this is part of why. I think we need to balance and draw in people from all walks - we want the clinicians to read this and see how great what we have done is without losing them on technical aspects. We want technical people to be drawn in to see that they can help solve critical problems, and we want advocates to see that we are using this data openly and to further research for all kids.